jsonschema/_utils.py

```diff
@@ -38,6 +38,22 @@ def __repr__(self):
         return repr(self.store)
 
 
+class Cache(object):
```

Comment:
Maybe a similar solution to what we do for remote cache would work, with the `cache` just being an optional argument which constructs an appropriately configured `lrucache`, but disabling caching would be providing a `functools.lrucache` with maxsize=0 or whatever.

But yeah, sounds like a decent idea. Nice catch.